core/lib/libtomcrypt/rsa.c

```diff
@@ -84,6 +85,10 @@ static TEE_Result tee_algo_to_ltc_hashindex(uint32_t algo, int *ltc_hashindex)
 
 TEE_Result crypto_acipher_alloc_rsa_keypair(struct rsa_keypair *s,
 					    size_t key_size_bits __unused)
+__attribute__((weak, alias("sw_crypto_acipher_alloc_rsa_keypair")));
```

Comment:
Since we already have a macro for weak functions (`__weak`), I'd prefer to define `__alias("foo")` in `compiler.h` so that this line would become `__weak __alias("sw_crypto_acipher_alloc_rsa_keypair")`

Reply:
why not just __weak_alias("foo") ?

Reply:
Fine with me